THIRD-PARTY-NOTICES

```diff
@@ -32,7 +32,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 The following NPM package may be included in this product:
 
- - @reduxjs/toolkit@1.6.0
+ - @reduxjs/toolkit@1.7.0
```

Comment:
Should we update the version in the package.json to this?
Seeing this made me think we may as well just update all our deps? only redux-thunk and redux-toolkit are non-dev dependencies
![image](https://user-images.githubusercontent.com/23005393/146042917-c7f79a29-5aaa-4c6d-bd52-bc01a997778d.png)

Reply:
this is the package I used https://www.npmjs.com/package/npm-check-updates

Reply:
updated!